src/test/java/io/cdap/plugin/WindowsAggregationUtilTest.java

```diff
@@ -139,14 +144,14 @@ public void setUp() {
     Mockito.when(sparkExecutionPluginContext.getFailureCollector()).thenReturn(failureCollector);
     sparkContext = Mockito.mock(SparkContext.class);
     Mockito.when(javaSparkContext.sc()).thenReturn(sparkContext);
-    List<Field> schemaFields = new ArrayList<>();
+    List<Schema.Field> schemaFields = new ArrayList<>();
```

Comment:
Why this change?

Reply:
Initially there was another Field class, due to which specifically it was mentioned.Removed it